packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+import {View} from "wonder-blocks-core";
+import {HeadingMedium} from "wonder-blocks-typography";
+
+type Props = {
+    /**
+     * The content of the modal, appearing between the header and footer.
+     */
+    content: React.Node,
+
+    /**
+     * The title of the modal, appearing in the header.
+     *
+     * If a subtitle is also present, this becomes smaller to accommodate both
+     * within the title bar.
+     */
+    title: string,
+
+    /**
+     * The subtitle of the modal, appearing in the header, below the title.
+     */
+    subtitle?: string,
+
+    /**
+     * The content of the modal's footer. A great place for buttons!
+     *
+     * Content is right-aligned by default. To control alignment yourself,
+     * provide a container element with 100% width.
+     */
+    footer?: React.Node,
+};
+
+/**
+ * The "standard" modal layout: a header, a content area, and a footer.
+ */
+export default class StandardModal extends React.Component<Props> {
+    render() {
+        return (
+            <View style={styles.container}>
+                <View style={styles.header}>
+                    <HeadingMedium style={styles.title}>
+                        {this.props.title}
+                    </HeadingMedium>
```

Comment:
We should also render the subtitle if it exists and render the title to be smaller in that case.

Reply:
haha totally forgot about subtitle partway through